app.py

```diff
@@ -3,3 +3,10 @@
 class MyClass:
     def __init__(self, value) -> None:
         self.value = value
+        self.another_value = 1
+
+    def print_value(self):
+        print(self.value)
```

Comment:
You shouldn't print inside the method, instead return the value
```suggestion
    def print_value(self):
        return self.value
```